CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to infiniboard
+
+Thanks for donating your time to developing infiniboard! This is the most
+valuable contribution you can make.
+
+This guide gives an overview of infiniboard and some development guidelines
+to make it easier for new contributors to get up and running. If you have
+any questions, feel free to get in touch.
+
+## Overview of infiniboard components
+Major components are documented via a README file in the folder of the component.
+
+## How can I contribute?
+
+Any kind of contribution is welcome as a pull request.
+If you are unsure on how to fork a repository and then create a pull 
+request from your fork, please read [this blog post](http://www.reflectoring.io/hacks/github-fork-and-pull/)
+for a quick guide.
+
+### Pick an Issue
+You can pick any issue from the [issue tracker](https://github.com/reflectoring/infiniboard/issues). 
+I try to document all feature ideas, tasks and bugs as issues in the tracker.
+Issues marked with the tag ["up-for-grabs"](https://github.com/reflectoring/infiniboard/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs) are especially documented so that 
+new contributors should get an idea of what is to do. You might want to
+start with one of those tasks. You can just as well pick any other task, though.
+If you have any questions, get in touch.
+
+### Submit an Idea
+If you have an idea, submit it to the [issue tracker](https://github.com/reflectoring/infiniboard/issues)
+for discussion. I will let you know my thoughts. If you want to develop your idea
+yourself, you can do so and submit a pull request.
+
```

Comment:
We have a wiki-page for discussions and ideas. Maybe we should provide the link in this guide too.